ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -2692,6 +2692,12 @@ impl<'a> Assemble<'a> {
             };
             self.header_start_locs.push(loc);
         }
+        // 16-byte align this jump target to improve performance.
+        let off = self.asm.offset().0;
+        let align = off.next_multiple_of(16);
```

Comment:
If you are already aligned, do we insert 16 nops?

Reply:
Good question. Let me check that.

Reply:
It doesn't. There's even an example in the docs: https://doc.rust-lang.org/std/primitive.u32.html#method.next_multiple_of

Reply:
Yep, `next_multiple_of` does exactly what we want!